Eastern-Research-Group/mywaterway
# Feature/538 surrounding cyan waterbodies

## Related Issues:
* [HMW-538](https://jira.epa.gov/browse/HMW-538)

## Main Changes:
* Added the CyAN layer to the Surrounding Features widget.
* Added the CyAN layer to the legend.

## Steps To Test:
1. Navigate to http://localhost:3000/community/lake%20okeechobee/monitoring.
2. Toggle on the _CyAN Waterbodies_ layer in the Surrounding Features widget.
3. Pan to a cyan waterbody outside the HUC boundaries, and click it to open a popup.
4. Verify that all features in the popup work as expected (especially the satellite imagery).
5. Click the waterbody inside the HUC boundaries.
6. Verify the satellite imagery is cleared from the previous waterbody.
7. Confirm the legend is updated when a cyan layer is visible, and that the symbol is not duplicated when both cyan layers are visible.
